src/pages/Home/components/BookSlider.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Slider from "react-slick";
+import PropTypes from 'prop-types';
+import "./BookSlider.css";
+
+const SampleNextArrow = (props) => {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={`${className} custom-arrow custom-next-arrow`}
+      style={{ ...style, display: 'block' }}
+      onClick={onClick}
+    />
+  );
+}
+
+const SamplePrevArrow = (props) => {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={`${className} custom-arrow custom-prev-arrow`}
+      style={{ ...style, display: 'block' }}
+      onClick={onClick}
+    />
+  );
+}
+
+const Book = ({ imageUrl }) => (
+  <div className="book-container rounded-lg overflow-hidden mx-1 transition-transform ease-in-out duration-300 hover:scale-110 cursor-pointer">
+    <img src={imageUrl} alt="Book cover" className="w-40 h-60 rounded-lg" />
+  </div>
+);
+
+const BookSlider = ({ books }) => {
```

Comment:
The name needs to be changed to Slider or Carousel etc. This will be used to list Chapter and Genre too. This name is too specific